test/src/ModuleTest.java

```diff
@@ -27,11 +45,11 @@ public void testNoModules() throws PrologException
 		{
 			System.err.println("Prolog compile error: " + error);
 		}
-		assertTrue(env.getLoadingErrors().size() == 0);
+                assertEquals(env.getLoadingErrors().size(), 0);
```

Comment:
The method signature for assertEquals is assertEquals(expected, actual) and you have actual, expected and so confusing error messages will be produced if this fails.